src/timedate/timedated.c

```diff
@@ -46,6 +46,7 @@
 #define NULL_ADJTIME_LOCAL "0.0 0 0\n0\nLOCAL\n"
 
 #define UNIT_LIST_DIRS (const char* const*) CONF_PATHS_STRV("systemd/ntp-units.d")
+#define SET_NTP_IN_FLIGHT_MAX 16
```

Comment:
this isn't really necessary btw. the bus broker puts a limit on in flight method calls anyway for each client, hence there's no real need to limits this here a second time.

Reply:
This is a limit on "server" side. It is meant to avoid exhaustion in timedated context caused by (multiple) client processes.
Should timedated capacity also considered indirectly limited by dbus capacity?